middleware/prometheus.go

```diff
@@ -119,3 +121,58 @@ func computeApproximateRequestSize(r *http.Request) int {
 	}
 	return s
 }
+
```

Comment:
I think this is wrong, also the written in delegator does not include the headers size.

Reply:
As we discussed response size metric is skipped for now. 

Later I figured out we could use messageview used in httplog to compute response size if we need it right away. But that would be just a hack (with several drawbacks). It's definitely not the proper way of doing it.